### PR TITLE
De-duplicate code around PairedData::ActiveInvocationTimes calls

### PR DESCRIPTION
**Pull request by akopich**
MizarPairedData now provider ActiveInvocationTimeSctats method

Tets: Unit, Manual
Bug: http://b/242839338

### CODE REVIEW

File: src/MizarData/MizarPairedDataTest.cpp [hunk 1]
```diff
@@ -253,33 +264,40 @@ TEST_F(MizarPairedDataTest, ForeachCallstackIsCorrect) {
   EXPECT_THAT(actual_ids_fed_to_action, UnorderedElementsAre(kAnotherCompleteCallstackIds));
 }
 
+const auto kExpectedInvocationTimes = {Times(kSamplingPeriod, uint64_t{2}),
```
Comment (karupayun) [inline src/MizarData/MizarPairedDataTest.cpp:267]: Nit: It's needed to wrap the 2 in a uint64_t?

**Review by karupayun — APPROVED** [review]
LGTM Valeriy! Just 2 nits.

File: src/MizarData/BaselineAndComparisonTest.cpp [hunk 2]
```diff
@@ -179,10 +181,17 @@ class MockFunctionTimeComparator {
   return result;
 }
 
+static void ExpectScopeStatsEq(const orbit_client_data::ScopeStats a,
+                               const orbit_client_data::ScopeStats b) {
+  EXPECT_EQ(a.ComputeAverageTimeNs(), b.ComputeAverageTimeNs());
+  constexpr double kTolerance = 1e-3;
```
[the remark this answers is not in the document]
Comment (akopich) [reply src/MizarData/BaselineAndComparisonTest.cpp:187]: Actually, we don't do any arithmetics in this test, we only need to make sure we get s copy of whatever the mock supplies. So... let's just have `EXPECT_EQ`

File: src/MizarData/MizarPairedDataTest.cpp (hunk 1 above)
Comment (akopich) [reply src/MizarData/MizarPairedDataTest.cpp:267]: Yeah, it's a a limitation of typedef arithmetics. We've declared multiply-ability with uint64_t and we have to supply it.

**Pull request merged**